db/migrate/20160805171111_create_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTopics < ActiveRecord::Migration
+  def change
+    create_table :topics do |t|
+      t.string :name
```

Comment:
when adding presence validations add a NOT NULL constraint here in order to be consistent